rasa/core/policies/ted_policy.py

```diff
@@ -65,14 +65,26 @@
     BALANCED,
     TENSORBOARD_LOG_DIR,
     TENSORBOARD_LOG_LEVEL,
+    ENCODING_LAYER_SIZE,
+    UNIDIRECTIONAL_ENCODER,
+    SEQUENCE,
+    SENTENCE,
 )
 
 
+if typing.TYPE_CHECKING:
+    from rasa.utils.features import Features
+
+
 logger = logging.getLogger(__name__)
 
-DIALOGUE_FEATURES = f"{DIALOGUE}_features"
-LABEL_FEATURES = f"{LABEL}_features"
-LABEL_IDS = f"{LABEL}_ids"
+MASK = "mask"
+LABEL_KEY = LABEL
```

Comment:
can we use `LABEL` directly?

Reply:
i prefer `LABEL_KEY` as it fits more to the data structure. Also aligns with `LABEL_SUB_KEY`. But if you have a strong argument for `LABEL` we can also change it.

Reply:
no strong argument, was just a bit confused by constant duplication